.github/workflows/collector.yml

```
@@ -168,15 +175,19 @@ jobs:
     - build-collector-image-remote-vm
     name: Create Multiarch manifest
     runs-on: ubuntu-24.04
```

CI is still skipping this because the `build-collector-image-remote-vm` is being skipped. Add this to force it to run:
```suggestion
    runs-on: ubuntu-24.04
    if: always()
```

---

Or you can refine it a bit more so it doesn't run with failures with something like this:
```suggestion
    runs-on: ubuntu-24.04
    if: always() && !contains(join(needs.*.result, ','), 'failure')
```